examples/src/update_doc/update_doc.go

```diff
@@ -19,7 +19,6 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-
```

Comment:
The space between stdlib imports and 3rd party imports is a proper import form, so let's put this empty line back.

Reply:
done in [a2a2c71](https://github.com/IBM/cloudant-go-sdk/pull/253/commits/a2a2c7123286ad912f29152e1dc093c4df9be1b3)